src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -87,6 +88,16 @@ const Withdraw = () => {
     },
   });
 
+  const {
+    data: emilyLimits,
+    refetch,
+    isLoading: emilyLimitsLoading,
+  } = useEmilyLimits();
+
+  const maxWithdrawal = (emilyLimits?.perWithdrawalCap || 0) / 1e8;
```

Comment:
a null `perWithdrawalCap` is inf, also didn't we already validate this `getEmilyLimits`? Maybe we can drop the nullable

Reply:
since this is a promise it will be nullable until it's resolved

Reply:
This seems a bit strange: should we use `useMemo` or something else, to ensure we compute `maxWithdrawal` only when `emilyLimits` is ready?

Reply:
we compute this even more frequently on every render some memos aren't that important the memory fingerprint is negligible